### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Dashboard_Databases_Screenreader : Table header is not announced by NVDA.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/azuredatastudio#10648]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#MAS1.3.1;#SQL Azure Data Studio;#FTP;#NVDA;#STP;#JAWS;#MAS2.4.6;#NonBenchmark;#Win32;#Acc123;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider (system setup)
Commit: 8288360
Date: 2020-05-29T05:54:49.731Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS1.3.1, MAS2.4.6
Screenreader : NVDA 2019.3.1,JAWS

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Double click on connected server or right click on it & select manage option to open the Dashboard.
4. Navigate to Databases under Dashboard & hit enter.
5.Navigate to databases table.
6.Start Screenreader as NVDA and observe whether NVDA is reading table headers or not. 

**Actual:**
NVDA is not reading the Databases table column header when we are navigating using up/down arrow key.

**Similar issue also observed with JAWS.**

**Similar issue also observed in following scenarios:**
1)Import Flat File Wizard-->Preview Data
2)Import Flat File Wizard-->Modify Columns
3)Import Flat File Wizard-->Summary

**Expected:**
NVDA should read the Databases table column header when we are navigating using up/down arrow key.

**User Impact:**
Screen reader users will not get information about the table column header.

**Attachment link for Reference:**

[10648_A11y_AzureDataStudioStayHealthy_Dashboard_Databases_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4707196/10648_A11y_AzureDataStudioStayHealthy_Dashboard_Databases_Screenreader.zip)

**Comment by alanrenmsft** [comment]
it is by design that the focus doesn't go to the table header, we have this pattern all of ADS, for example query editor's query result table. the header information is announced when you are navigating inside the row using tab or left/right keys, see my screenshot below
![Screen Shot 2020-07-13 at 7 15 55 PM](https://user-images.githubusercontent.com/13777222/87372902-81d33180-c53d-11ea-82bd-5b7a61f23869.png)

**Comment by Rama8345** [comment]
Verified the issue at Azure Data Studio. the issue is not reproducible. So, closing the bug. Please find attachment for reference.
**Github Tags:**#Closed;
![10648_Closed](https://user-images.githubusercontent.com/115216656/221556406-51eb976c-5ecd-4eee-9ea0-d904acf6b5e3.gif)
